projects/packages/forms/src/modules/form-step/view.js

```diff
@@ -0,0 +1,21 @@
+import { getContext, store } from '@wordpress/interactivity';
+
+const NAMESPACE = 'jetpack/form';
+store( NAMESPACE, {
```

Comment:
Do we need some ID-prefixing here to avoid having same state across multiple form instances on same page?